core/src/main/java/org/mule/runtime/core/internal/retry/policies/SimpleRetryPolicy.java

```diff
@@ -194,4 +197,17 @@ protected AtomicInteger initialValue() {
       return new AtomicInteger(0);
     }
   }
+
+  private void logRetrying(long attempts) {
+    if (LOGGER.isInfoEnabled()) {
+      LOGGER.info("Retrying execution of event, attempt {} of {}.", attempts,
+                  count != RETRY_COUNT_FOREVER ? valueOf(count) : "unlimited");
+    }
+  }
+
+  private void logRetriesExhausted() {
+    if (LOGGER.isInfoEnabled()) {
+      LOGGER.info("Retry attempts exhausted. Failing...");
+    }
+  }
```

Comment:
This could be very verbose, maybe change to DEBUG? what was the original reason for removing this in the first place?

Reply:
It wasn't removed.  This piece of code didn't change between `4.2.2` and `4.3.x`.  The retries are logged [here](https://github.com/mulesoft/mule/blob/support/4.2.x/core/src/main/java/org/mule/runtime/core/internal/retry/policies/SimpleRetryPolicy.java#L123) in INFO, but they don't [here](https://github.com/mulesoft/mule/blob/support/4.2.x/core/src/main/java/org/mule/runtime/core/internal/retry/policies/SimpleRetryPolicy.java#L72).

Reply:
I changed the logging level to DEBUG instead of INFO.